executor/builder.go

```diff
@@ -143,6 +145,19 @@ func (b *executorBuilder) build(p plan.Plan) Executor {
 	}
 }
 
+func (b *executorBuilder) buildCancelDDLJobs(v *plan.CancelDDLJobs) Executor {
+	e := &CancelDDLJobsExec{
+		baseExecutor: newBaseExecutor(v.Schema(), b.ctx),
+		JobIDs:       v.JobIDs,
+	}
+	e.errs, b.err = inspectkv.CancelJobs(e.ctx.Txn(), e.JobIDs)
```

Comment:
Why not do this in CancelDDLJobsExec.Next()?

Reply:
It seems `CancelDDLJobsExec.Next()` has no `txn`.

Reply:
We can new a `txn`  and commit this `txn` in `CancelDDLJobsExec.Next()`. But this solution will commit the previous transaction.